CHARTER.md

```diff
@@ -1,51 +1,123 @@
+
+Once rejected, a proposal may not be submitted again for 30 days.
+
+ART members cast their votes by reviewing the pull request. An approval vote is cast by
+approving the pull request, and a rejection vote is cast by using the *Request changes*
+function and providing feedback in the comment regarding the reason for the rejection
+and the changes they would like to see made. Once the voting period is over or all ART
+members have voted, the pull request is merged into the `main` branch (always, whether
+approved or rejected). The chairs then record the decision, dissenting comments, and
+other relevant information in the [`RECORD.md`](art-decisions/RECORD.md) file via
+another pull request that is reviewed and approved by two active chairs to ensure
+accuracy.
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+## Chair Elections
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the `art-decisions` folder. Silence equals abstaining from a vote.
+If a chair leaves through non-participation, through their own resignation, or due to an
+incapacity to carry out their duties, elections for a new chair take place.
 
-## Chair Elections
+After the announcement of a chair leaving, there is a period of 30 days for nominations
+(from the date of the announcement). Any ART member can nominate another ART member or
+themselves, while a second member has to sponsor the nomination. Only members that have
+been part of the ART for at least six months are eligible to become a chair or sponsor a
+nomination.
 
-If a chair leaves through non-participation or through their own announcement, elections for a new chair take place.
+After the nomination period, elections take place asynchronously through the same
+process as standard proposals.
 
-After the announcement of a chair leaving, there is a period of 30 days for nominations (from the date of the announcement). Any ART member can nominate another ART member or themselves, while a second member has to sponsor the nomination. Only members that have been part of the ART for 6+ months are eligible to become a chair or sponsor a nomination.
+## Code of Conduct Enforcement
 
-After the nomination period, elections take place asynchronously through the same process as standard decision proposals.
+Removing or disciplining an ART member due to a violation of the [Arcalot Code of
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is initiated
+by any chair as a pull request to the ART. Only the chairs vote on these matters, with
+an approval by unanimous consent of the active chairs and with a 7 calendar day voting
```

Comment:
Given that "Chairs are expected to remain actively involved", we probably should not use the phrase "active chairs" elsewhere in the text -- we should use just "chairs".  We do this in four places, currently.